src/store/TodoItem.ts

```diff
@@ -0,0 +1,14 @@
+type TodoItem = {
+  id?: string;
+  title: string;
+  description: string;
+  category: number;
+  dueDate?: string;
+  cardColor: number;
+  isImportant: boolean;
```

Comment:
I think it can be `nullable` field.

Reply:
It's boolean, by default all boolean is False. Do we need mark this variable as nullable ?

Reply:
Keep it